virtio-devices/src/vhost_user/fs.rs

```diff
@@ -356,18 +357,20 @@ impl Fs {
                 avail_features: acked_features,
                 acked_features: 0,
                 queue_sizes: vec![queue_size; num_queues],
-                paused_sync: Some(Arc::new(Barrier::new(2))),
+                paused_sync: Some(Arc::new(Barrier::new(3))),
```

Comment:
Maybe we need to change the 3 back to 2 in the last two commits of this PR, as we are merging the reconnect thread and the slave thread into one.

Reply:
Definitely. I'll fix this as well.